start_sshd.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
+
+[ -f "$XDG_RUNTIME_DIR/openssh-rs/sshd_started" ] && exit
+
+cd $(dirname `realpath $0`)
+
+# Start the container
+export PUBLIC_KEY='ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGzHvK2pKtSlZXP9tPYOOBb/xn0IiC9iLMS355AYUPC7'
+export DOCKER_MODS='linuxserver/mods:openssh-server-ssh-tunnel'
+
+docker run \
+    --name openssh \
+    --rm \
+    -d \
+    -p 127.0.0.1:2222:2222 \
+    -e 'USER_NAME=test-user' \
+    -e DOCKER_MODS \
+    -e PUBLIC_KEY \
+    linuxserver/openssh-server:amd64-latest
+
+export HOSTNAME=127.0.0.1
+chmod 600 .test-key
+
+# Remove 127.0.0.1:2222 from known_hosts
+ssh-keygen -R "[127.0.0.1]:2222"
+rm -f ~/.ssh/known_hosts.old
```

Comment:
Why remove this file too?

Reply:
`~/.ssh/known_hosts.old` sometimes cause the ssh integration test to fail.

Reply:
The solution here, will be to change the path to `known_hosts` file used by the integration tests to avoid any possible interference of the user's `known_hosts` file.

Reply:
#54 will provide a proper solution to this problem.

Reply:
#54 has been merged :tada:

Reply:
Done!